be/src/storage/tablet_schema.cpp

```diff
@@ -187,6 +187,48 @@ std::string TabletColumn::get_string_by_aggregation_type(FieldAggregationMethod
     return "";
 }
 
+size_t TabletColumn::estimate_field_size(size_t variable_length) const {
```

Comment:
You can define this function in storage/type_utils.h

Reply:
Fixed